modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -261,9 +284,16 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
             $where        .= " AND s.Visit_label = :VL";
             $qparams['VL'] = $visit_label;
         }
-        if (($site !=null)&&($site!='All Sites')) {
-            $where         .= " AND s.CenterId = :CID";
-            $qparams['CID'] = $site;
+        if ($site!='All Sites') {
+            if ($site == 'All User Sites') {
+                $user           = User::singleton();
+                $siteID         = $user->getCenterIDs();
+                $where         .= " AND FIND_IN_SET(psc.CenterID, :CID)";
+                $qparams['CID'] = implode(',', $siteID);
+            } else {
```

Comment:
@sruthymathew123 
I am not sure the purpose of this `else` here.
User is at either all sites or at own sites, whatelse should we consider?